easybuild/easyconfigs/m/MariaDB/MariaDB-10.3.14-foss-2019a.eb

```diff
@@ -0,0 +1,62 @@
+easyblock = 'CMakeMake'
+
+name = 'MariaDB'
+version = '10.3.14'
+
+homepage = 'https://mariadb.org/'
+description = """MariaDB is an enhanced, drop-in replacement for MySQL.
+Included engines: myISAM, Aria, InnoDB, RocksDB, TokuDB, OQGraph, Mroonga."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = ['http://ftp.hosteurope.de/mirror/archive.mariadb.org/mariadb-%(version)s/source']
+sources = [SOURCELOWER_TAR_GZ]
```

Comment:
are the submodules (in the `10.3.10` easyconfig) no longer necessary?

Reply:
Both MariaDB Connector/C and RocksDB engine are included in the tarball.